mk/spksrc.archs.mk

```diff
@@ -6,7 +6,9 @@
 # SRM - Synology Router Manager
 SRM_ARMv7_ARCHS = northstarplus ipq806x dakota
 # DSM - all ARMv7 except SRM specific archs
-DSM_ARMv7_ARCHS = alpine armada370 armada375 armada38x armadaxp comcerto2k monaco
+DSM_ARMv7_ARCHS = alpine armada370 armada375 armada38x armadaxp monaco
+# comcerto2k is the only ARMv7 arch that uses an GCC (4.9.3) and GLIBC (2.20)
+OLD_DSM_ARMv7_ARCHS = comcerto2k
```

Comment:
not sure, whether we need this, as comcerto2k has to be excluded from generic armv7 for DSM7 only and is not used to define specific configure/make configs or to define unsupported archs.
_anyway it will be better to define unsupported packages based on gcc/kernel or other versions and not on arch and DSM version_